app/models/group.js

```diff
@@ -181,5 +187,24 @@ exports.addModel = function(database) {
     })
   }
 
+  /**
+   * Checks if the specified user can post to the timeline of this group.
+   */
+  Group.prototype.validateCanPost = function(postingUser) {
```

Comment:
Yes, it'd be nice to have a more general solution here. However, I usually prefer to generalize a solution once we have more than one specific case. :)